website/content/docs/ecs/compatibility.mdx

```diff
@@ -0,0 +1,24 @@
+---
+layout: docs
+page_title: Compatibility Matrix
+description: Compatibility Matrix for Consul ECS
+---
+
+# Compatibility Matrix for Consul on ECS
+
+For every release of Consul on ECS, `consul-ecs` binary and `consul-ecs` terraform module is updated. The terraform module and binary are tightly coupled. Ex: consul-ecs binary 0.4.1 must use consul-ecs terraform 0.4.1.
+
+## Supported Consul versions
+
+### Version 0.2.0 and above
+
+
+| Consul Version | Compatible consul-ecs Versions  |
+| -------------- | ------------------------------- |
+| 1.10.x         | 0.2.x                           |
+| 1.11.x         | 0.3.0, 0.4.x                    |
```

Comment:
What about Consul version 1.12.x (now released)?

Reply:
Jared, 
We have not yet qualified the released version with latest version of Consul.

The plan for now is to qualify the upcoming release of Consul on ECS (0.5) with the latest of Consul core (1.12) at that time. In the future as Consul on ECS gains more traction, we will add test framework to qualify existing releases (0.4, 0.5,..) with latest core (1.13, 1.14..). Right now focusing on feature velocity.

Reply:
I can imagine a reader wondering: Is 1.12.x not listed because this page is just out of date? Or is it because 1.12.x actually isn't tested yet?

Would it make sense to have a row for Consul 1.12.x where the compatible consul-ecs version says "Not available yet"?

Reply:
That would be a good idea. Will update the PR!